bril-txt/briltxt.py

```diff
@@ -170,9 +261,17 @@ def print_prog(prog):
 
 # Command-line entry points.
 
-def bril2json():
-    print(parse_bril(sys.stdin.read()))
+def linkbril():
+    main_module = sys.argv[1]
+    module_paths = sys.argv[2:]
+    json = link_bril(main_module, module_paths)
+    if json and json != 'null':
+        print(json)
 
+def bril2json():
+    json = parse_bril(sys.stdin.read())
+    if json and json != 'null':
```

Comment:
If `json == 'null'` that means something went wrong. Should figure out why this is happening.